src/types/index.ts

```diff
@@ -10,7 +10,7 @@ import {
 
 export type ReactChildren = React.ReactNode;
 
-export type ToastType = string;
+export type ToastType = 'success' | 'error' | 'info';
```

Comment:
```suggestion
export type ToastType = 'success' | 'error' | 'info' | (string & {});
```
to allow other strings as well (there can be custom toast types). 

`(string & {})` is [a workaround](https://stackoverflow.com/a/61048124), to keep the type wide, but also provide auto-complete suggestions in the editor